frontend/src/scenes/insights/views/WorldMap/worldMapLogic.tsx

```diff
@@ -8,9 +8,9 @@ export const worldMapLogic = kea<worldMapLogicType>({
     props: {} as InsightLogicProps,
     key: keyForInsightLogicProps('new'),
     path: (key) => ['scenes', 'insights', 'WorldMap', 'worldMapLogic', key],
-    connect: {
+    connect: () => ({
         values: [insightLogic, ['insight', 'filters']],
```

Comment:
Should it be passing props to choose a specific `insightLogic` here? Is it assuming it is inside a `<BindLogic/>`

Reply:
Oh, there's a slightly unexpected/magical behaviour at play here: `props` are passed along if you don't specify them explicitly during `connect`. 🙃 🙈

Reply:
![](https://c.tenor.com/wn2_Qq6flogAAAAC/magical-magic.gif)